website/docs/commands/misc/config.md

```diff
@@ -77,10 +77,10 @@ scala-cli --power config --password github.token
 
 </ChainedSnippets>
 
-Use `--create-key` to create a PGP key pair, protected by a randomly-generated password, to
+Use `--create-pgp-key` to create a PGP key pair, protected by a randomly-generated password, to
 be used by the `publish setup` sub-command:
-```sh
-scala-cli config --create-key
+```bash
+scala-cli --power config --create-pgp-key
```

Comment:
The same situation as in #1898, maybe we can change config directory by default for `scalaclicheck` module. It could be handled around [here](https://github.com/VirtusLab/scala-cli/blob/0d4c5f762010c13172b5c413f662f63cfa66c803/build.sc#L117).